### PR TITLE
Fix: Do not use `ant` on GitHub Actions

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] stops using `ant` on GitHub Actions

### CODE REVIEW

File: .github/workflows/integrate.yaml [hunk 1]
```diff
@@ -29,29 +29,29 @@ jobs:
...
+          wget https://phar.io/releases/phive.phar
+          wget https://phar.io/releases/phive.phar.asc
+          gpg gpg-get-key --recv-keys 0x9D8A98B29B2D5D79
+          gpg --verify phive.phar.asc phive.phar
+          chmod +x phive.phar
+          rm phive.phar.asc
 
       - name: "Install dependencies with phive"
-        run: "ant -Dphive.bin=./phive.phar install-tools"
+        run: "phive install --copy --trust-gpg-keys 4AA394086372C20A,2A8299CE842DD38C,CF1A108D0E7AE720,2DE50EB60C013FFFA831040D12CE0F1D262429A5,E82B2FB314E9906E"
```
Comment (theseer) [inline .github/workflows/integrate.yaml:45]: How does this call succeed? The download above stores it as `phive.phar`. Either it's wrong or the wget lines are useless ;)

File: .github/workflows/integrate.yaml [hunk 2]
```diff
@@ -28,30 +28,22 @@ jobs:
         uses: "shivammathur/setup-php@v2"
         with:
           php-version: "${{ matrix.php-version }}"
-
-      - name: "Set up Java"
-        uses: "actions/setup-java@v3"
-        with:
-          distribution: "zulu"
-          java-version: "11"
+          tools: "phive"
```
Comment (localheinz) [inline .github/workflows/integrate.yaml:31]: Would that be an option for you, @theseer?

Comment (theseer) [reply .github/workflows/integrate.yaml:31]: If it works ;)

And depending on how it it installs it ;)

**Comment by theseer** [comment]
Is this still relevant? :-)

**Comment by theseer** [comment]
@localheinz ping? :-)

**Comment by localheinz** [comment]
@theseer 

I have just rebased - merge or close at will!

**Pull request merged**